vnext/Microsoft.ReactNative.Cxx/DesktopWindowBridge.h

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <ReactNotificationService.h>
+#include <ReactPropertyBag.h>
+#include <winrt/Windows.Foundation.h>
+#include <functional>
+
+#if defined(USE_WINUI3) && !defined(CORE_ABI) && !defined(__APPLE__) && \
+    WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_APP | WINAPI_PARTITION_SYSTEM | WINAPI_PARTITION_GAMES)
+
+#include <winrt/Microsoft.ReactNative.h>
+
+namespace Microsoft::ReactNative {
+
+inline auto GetPropertyNameForWindowMessage(UINT uMsg) {
+  auto id = std::to_wstring(uMsg);
+  winrt::Microsoft::ReactNative::ReactPropertyName windowMsgProperty{L"ReactNative.Desktop.WindowMessage", id.c_str()};
+  return windowMsgProperty;
+}
+
+inline void ForwardWindowMessage(
+    const winrt::Microsoft::ReactNative::IReactNotificationService &svc,
+    HWND hwnd,
+    UINT uMsg,
+    WPARAM wParam,
+    LPARAM lParam) {
+  winrt::Microsoft::ReactNative::DesktopWindowMessage args{uMsg, wParam, lParam};
+  svc.SendNotification(GetPropertyNameForWindowMessage(uMsg).Handle(), winrt::box_value((uint64_t)hwnd), args);
+}
+
+inline winrt::Microsoft::ReactNative::IReactNotificationSubscription SubscribeToWindowMessage(
+    const winrt::Microsoft::ReactNative::IReactNotificationService &svc,
+    UINT uMsg,
+    std::function<void(HWND, const winrt::Microsoft::ReactNative::DesktopWindowMessage &)> callback) {
```

Comment:
>> [](start = 89, length = 1)

It better to avoid passing by value types that have non-default constructor. It produces less efficient code. In this case is better to use r-value reference && to indicate that the code takes the ownership of the callback and then use std::move to capture it.